src/app/shared/code-editor-list/code-editor-list.component.ts

```diff
@@ -129,7 +129,7 @@ export class CodeEditorListComponent {
     } else if (this.fileType === 'testParam') {
       return '.json';
     } else if (this.fileType === 'dockerfile') {
-      return '/Dockerfile';
+      return 'Dockerfile';
```

Comment:
Is Dockerfile losing the leading slash on purpose?

Reply:
Yes, since it is added above in a more general case.